pom.xml

```diff
@@ -272,9 +272,14 @@
                 <version>2.2.5</version>
             </dependency>
             <dependency>
+                <groupId>org.glassfish.web</groupId>
+                <artifactId>javax.el</artifactId>
+                <version>2.2.6</version>
```

Comment:
spring-boot-starter-web includes tomcat-embed-el if tomcat is used (default).
I think some of the dependencies should not be included by default as they might conflict with whatever project uses.